packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -175,6 +175,17 @@ export class Tracer implements types.Tracer {
     this.eventListenersLocal.push(listner);
   }
 
+  /**
+   * Unregisters an end span event listener.
+   * @param listener The listener to unregister.
+   */
+  unregisterSpanEventListener(listner: types.SpanEventListener) {
```

Comment:
Nit: `listener` (and in other files as well).

Reply:
Thanks for noticing.